packages/orm/src/EntityManager.ts

```diff
@@ -474,9 +504,13 @@ export class EntityManager {
   /**
    * Flushes the SQL for any changed entities to the database.
    *
-   * Currently this `BEGIN`s and `COMMIT`s a new transaction on every call;
-   * we should also support an `EntityManager` itself running all queries
-   * (i.e. including the initial `SELECT`s) in a transaction.
+   * If this is run outside of an existing transaction, it will `BEGIN` and `COMMIT`
+   * a new transaction on every `.flush()` call so that all of the `INSERT`s/etc.
+   * happen atomically.
+   *
+   * This is it called with an existing transaction, i.e. `EntityManager.transaction`
```

Comment:
Think you just want "If this is called with an existing transaction"

Also should remove the reference to `newEntityManager` here since that seems to have gone away.

Reply:
Yep, will fix.